src/main/resources/template.properties

```diff
@@ -32,6 +32,22 @@ hivetobq.partition.col=
 # Spark warehouse directory location
 hivetobq.spark.sql.warehouse.dir=<spark-warehouse-directory>
 
+# JDBCToBQ Template properties.
+##Bigquery table name
+jdbctobq.bigquery.location=
+jdbctobq.jdbc.url=jdbc:mysql://host:port/
+jdbctobq.jdbc.driver.class.name=com.mysql.jdbc.Driver
+#Customize any properties supported by jdbc. Value to be in JSON format
+jdbctobq.jdbc.properties={"user":"usernamehere","password":"passwordhere"}
+##jdbc:bigquery://https://www.googleapis.com/bigquery/v2:443;ProjectId=nsadineni;OAuthType=1;
+jdbctobq.input.table=
+jdbctobq.input.db=
+#Write mode to use while writing output to BQ. Supported values are - Append/Overwrite/ErrorIfExists/Ignore
+jdbctobq.append.mode=ErrorIfExists
```

Comment:
is _append.mode_ the best name for this variable? Should it not be something like _write.mode_

Reply:
Thanks Frank. I have modified the variable name to write.mode